specification/monitor/resource-manager/microsoft.insights/2017-11-01-preview/baseline_API.json

```diff
@@ -0,0 +1,347 @@
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            },
+            "200": {
+              "description": "Successful request to get the list of metric values.",
+              "schema": {
+                "$ref": "#/definitions/BaselineResponse"
+              },
+              "examples": {
+                "application/json": {
+                  "id": "/subscriptions/b368ca2f-e298-46b7-b0ab-012281956afa/resourceGroups/vms/providers/Microsoft.Compute/virtualMachines/vm1/providers/Microsoft.Insights/baseline/PercentageCpu",
+                  "type": "Microsoft.Insights/baseline",
+                  "name": {
+                    "value": "PercentageCpu"
+                  },
+                  "properties": {
+                    "timespan": "2017-04-14T02:20:00Z/2017-04-14T04:20:00Z",
+                    "aggregation": "Average",
+                    "interval": "PT1H"
+                  },
+                  "timestamps": [
+                    "2017-04-14T02:20:00Z",
+                    "2017-04-14T03:20:00Z"
+                  ],
+                  "baseline": [
+                    {
+                      "sensitivity": "Low",
+                      "lowThresholds": [
+                        30.0,
+                        31.1
+                      ],
+                      "highThresholds": [
+                        90.3453,
+                        91.3453
+                      ]
+                    },
+                    {
+                      "sensitivity": "Medium",
+                      "lowThresholds": [
+                        50.0,
+                        51.1
+                      ],
+                      "highThresholds": [
+                        70.3453,
+                        71.3453
+                      ]
+                    }
+                  ]                  
+                }
+              }
+            }
+          },
+          "x-ms-examples": {
+            "Get Metric for data": { "$ref": "./examples/GetBaseline.json" },
+            "Get Metric for metadata": { "$ref": "./examples/GetBaselineMetadata.json" }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "LocalizableString": {
+        "required": [ "value" ],
+        "properties": {
+          "value": {
+            "type": "string",
+            "description": "the invariant value."
+          },
+          "localizedValue": {
+            "type": "string",
+            "description": "the locale specific value."
+          }
+        },
+        "description": "The localizable string class."
+      },
+      "BaselineProperties": {
+        "properties": {
+          "timespan": {
```

Comment:
date-time format?

Reply:
It should be an ISO 8601 time interval. The format is "{date-time}/{date-time}". So string is probably more appropriate here.
I tried to be consistent with the definitions and types in the metrics_API.